build.py

```diff
@@ -525,6 +528,9 @@ def _likely_binary(sdist: str) -> str | None:
         if "/test/" in name or "/tests/" in name:
             continue
 
+        if any(substr in name for substr in package.likely_binary_ignore):
```

Comment:
I think these would be better as exact matches?

Reply:
ok!